src/routers/get.js

```diff
@@ -0,0 +1,36 @@
+const { FastifyPluginCallback } = require("fastify");
+
+const store = require("../store");
+const errHandler = (err) => {console.log(err);}
+
+/**
+ * @type {FastifyPluginCallback}
+ */
+
+const getPlugin = (fastify, opts, done) => {
+    fastify.get("/:first_id", async (req, reply) => {
```

Comment:
while this approach for get requests does cut down of lines of code, I wouldn't reccommend it as it is not explicit about what is being done. should stick to GET /courses/{id}, /outcomes/{id} or something like GET /pillars/{id}/courses/{id}/outcomes/{id} to make the relationship between pillars, courses and outcomes explicit.